Dockerfile

```diff
@@ -443,7 +443,7 @@ RUN mkdir -p /opt/php/bin \
 # nuget-inspector
 ENV NUGET_INSPECTOR_HOME=/opt/nuget-inspector/bin
 ENV PATH=$PATH:$NUGET_INSPECTOR_HOME
-ARG NUGET_INSPECTOR_VERSION=0.9.1
+ARG NUGET_INSPECTOR_VERSION=0.9.6
```

Comment:
We saw everything being moved to plugins, so we followed this. Are you saying we should revert this and not use plugins after all?

Reply:
> We saw everything being moved to plugins

It's not "everything" (yet). The work is centrally tracked at https://github.com/oss-review-toolkit/ort/issues/6602, which mentions that the DotNet / NuGet moved should be delayed until this PR is merged.

> Are you saying we should revert this and not use plugins after all?

Now that the work has already been done, it could also make sense to keep it. I need to discuss this with @MarcelBochtler. Please don't take any actions in that regard until then.